Makefile

```diff
@@ -19,32 +19,35 @@ endif
 .PHONY: init
 # init env
 init:
-	go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
-	go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
-	go install github.com/go-kratos/kratos/cmd/kratos/v2@latest
```

Comment:
Kratos need to be present

Reply:
My bad, meant to only remove protoc plugin downloads.